server/views/admin/users/index.njk

```diff
@@ -4,69 +4,113 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/pagination/macro.njk" import govukPagination %}
-
-{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% set pageTitle = applicationName + " - " + pageHeading %}
 {% set mainClasses = "app-container govuk-body assessments--index" %}
-
-{% block beforeContent %}
-    {{ govukBackLink({
-		text: "Back",
-		href: '/'
-	}) }}
-{% endblock %}
-
+{% block beforeContent %}{{ govukBackLink({"text": "Back", "href": "/"}) }}{% endblock %}
```

Comment:
It's hard to work out what are formatting changes and what are additions to the template. 
I suggest using https://marketplace.visualstudio.com/items?itemName=okitavera.vscode-nunjucks-formatter to format the template or an equivalent for your IDE.  
If formatting changes need to be made please do this in a seperate commit before adding the new code so it's easier to read.